recipes/scaden/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "scaden" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c5c3c58aa9a9c36151dcbd746445ed038fbc33e7906529a2bbff1d84dcfb7239
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+    - twine
+  run:
+    - python
+    - pandas ==0.21
+    - numpy
+    - scikit-learn
+    - scipy
+    - seaborn
+    - tensorflow ==1.10.0
+    - matplotlib
+    - scanpy
+    - tqdm
+    - click
+
+about:
+  home: https://github.com/KevinMenden/scaden
+  license: MIT
+  summary: Cell type deconvolution using single cell data
+
+extra:
+  skip-lints:
+    - missing_tests
```

Comment:
- Add a comment explaining that tests are disabled because of tensor flow problem. 
- Also add some tests that work for you locally (commented out), so that we have something to enable once the tensorflow libc problem has been fixed.
- Would appreciate if you could add yourself as maintainer.

Reply:
Added a comment about that in new commit.
Also added one test that at least locally worked (scaden --help).

Reply:
Personally, I think the `--help` type test is completely insufficient. But then we have a lot of precedent for those. ;)